infra/cifuzz/config_utils.py

```diff
@@ -209,6 +209,7 @@ def __init__(self):
     self.git_store_branch = os.environ.get('GIT_STORE_BRANCH')
     self.git_store_branch_coverage = os.environ.get('GIT_STORE_BRANCH_COVERAGE',
                                                     self.git_store_branch)
+    self.upload_build = os.getenv('UPLOAD_BUILD')
```

Comment:
Probably better to use `environment.get_bool('UPLOAD_BUILD')`

Reply:
done!